src/backend/common/tasks/handlers/task_handler.py

```diff
@@ -0,0 +1,46 @@
+import pickle
+from typing import Any
+
+from flask import Flask, request, Response
+from werkzeug.routing import BaseConverter
+
+
+class RegexConverter(BaseConverter):
+    def __init__(self, url_map, *items) -> None:
+        super(RegexConverter, self).__init__(url_map)
+        self.regex = items[0]
+
+
+class PermanentTaskFailure(Exception):
+    """Indicates that a task failed, and will never succeed."""
+
+
+def run(data: bytes) -> Any:
+    """Unpickles and executes a task.
+
+    Args:
+        data: A pickled tuple of (function, args, kwargs) to execute.
+    Returns:
+        The return value of the function invocation.
+    """
+    try:
+        func, args, kwargs = pickle.loads(data)
+    except Exception as e:
+        raise PermanentTaskFailure(e)
+    else:
+        return func(*args, **kwargs)
+
+
+def handle_defer(path: str) -> Response:
+    # TODO: Test to make sure we don't need any sort of auth mechanism here
+    run(request.get_data())
+    return Response(status=200)
+
+
+def install_defer_routes(app: Flask) -> None:
+    app.url_map.converters["regex"] = RegexConverter
+    app.add_url_rule(
+        '/_ah/queue/<regex("deferred.*"):path>',
```

Comment:
Note to Zach: This can't have a prefix, despite the work we're doing with prefix routing. All of the existing GAE stuff assumes we'll be hitting some `/_ah/queue/deferred.*` path upstream, not `/prefix/_ah/queue/deferred.*`. This slightly complicates things because it means we probably shouldn't use our dispatcher as a form of routing. We need to hit services proper. Locally, this will probably mean using ports?